sqlmesh/lsp/main.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
```

Comment:
Do we need this?

Reply:
We don't ... but it allows for using running the script like 

```shell
./sqlmesh/lsp/main.py
```